plugin/PlayAssetDelivery/build.gradle

```diff
@@ -20,13 +20,20 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+
+    testOptions {
```

Comment:
This line is added since SignalInfo is depended on android.text.TextUtils, and we need to solve [this](https://stackoverflow.com/questions/35763289/need-help-to-write-a-unit-test-using-mockito-and-junit4) issue.